randnum.c

```diff
@@ -0,0 +1,25 @@
+//Olivier Gabison - Random number generator that checks whether you guessed correctly.
+
+#import <stdio.h>
```

Comment:
Again, you must have been coding in Python too recently before doing this assignment. :) Aside from this, solid code though!